cats/forecast.py

```diff
@@ -1,5 +1,5 @@
 from dataclasses import dataclass, field
-from datetime import datetime
+from datetime import datetime, timedelta
 
 
 @dataclass(order=True)
```

Comment:
Also not a new change, but on the `CarbonIntensityPointEstimate`, I think the `sort_index` introduces unnecessary code. As discussed previously and [here](https://stackoverflow.com/questions/68840767/what-is-the-purpose-of-sort-index-of-a-dataclass-in-python), the first attribute is used for sorting. I'd suggest removing sort_index and just putting a comment there for clarity, something like below. Same thing for `CarbonIntensityAverageEstimate`.

```python
class CarbonIntensityPointEstimate:
    """Represents a single data point within an intensity
    timeseries. Use order=True in order to enable comparison of class
    instance based on the first attribute.  See
    https://peps.python.org/pep-0557

    """
    value: float # the first attribute is used automatically for sorting methods
    datetime: datetime
    
```

Reply:
Agreed! I won't add that to that PR, but feel free to make that change of you happen to have some time for it

Reply:
Addressed with #58